tests/codec/ber/test_decoder.py

```diff
@@ -1669,8 +1658,11 @@ def testInvalidFileContent(self):
                 out.write(ints2octs((2, 1, 12, 35, 128, 3, 2, 0, 169, 3, 2, 1, 138, 0, 0, 7)))
 
             with open(path, "rb") as source:
-                with self.assertRaises(SubstrateUnderrunError):
-                    _ = list(decoder.decodeStream(source))
+                list(decoder.decodeStream(source))
+
+        except EndOfStreamError:
```

Comment:
Is it possible to use `assertRaises`? If we expect an exception, we should check that it really was raised.

Reply:
> Is it possible to use `assertRaises`? If we expect an exception, we should check that it really was raised.

Sure! I feel like we should first convert all legacy `assert` statements cross the whole unit test code into `self.assert*` (as part of deprecating older Pythons). Just for sake of uniformity.